### ISSUE TITLE
Preparing for v1.4 release

### ISSUE THREAD

**Issue by rth** [bsmurphy/PyKrige#67]
As discussed in #60, there were quite a bit of new features added since last year, and it would be good to make a new release (v1.4).

@bsmurphy @basaks What do you think is left to do to make this release happen? Could you please add / remove open issues to/from the [v1.4 milestone](https://github.com/bsmurphy/PyKrige/milestone/2) if needed? Thanks!

**Comment by bsmurphy** [comment]
Sorry I've been so slow on this, got a lot going on right now... I'm hoping to have some time later this week to work on this, so I'll adjust the v1.4 milestones and lay out a clearly plan here once I've put more thought into it...

**Comment by bsmurphy** [comment]
OK finally setting aside a solid chunk of time to work on this. For the v1.4 milestone, the only thing really remaining that we planned on was the docstring restructuring (#65). Though we had talked about slowly transitioning that over, with the docs online maybe it would be better to just push out some reorganization right now before releasing v1.4? What do you think @rth? Would be good to get v1.4 pushed out ASAP that way we can update PyPI with a clean new version.

**Comment by rth** [comment]
> with the docs online maybe it would be better to just push out some reorganization right now before releasing v1.4? 

Yes, I agree that it might be easier to just make those changes in one PR. Right now the API docs don't look so good (e.g. this [documentation](http://pykrige.readthedocs.io/en/latest/generated/pykrige.uk.UniversalKriging.html#pykrige.uk.UniversalKriging)), it would be useful make them better for v1.4 (and it wouldn't take too much effort I think). 

On my side, I have started refactoring of unit tests to use pytest in https://github.com/bsmurphy/PyKrige/pull/80 . I still need to figure out why they buid fails; but I'll try to get to the bottom of it this weekend.

In any case, it would be great to have v1.4 out there!

**Comment by bsmurphy** [comment]
Cool, I'll work on the docstrings before anything else.

**Comment by bsmurphy** [comment]
Oh, also, I disabled CircleCI builds so we can remove that stuff

**Comment by bsmurphy** [comment]
Alright, so we've dealt with the testing stuff and the docs reformatting, anything else we need to do for v1.4? Only thing I can think of is updating the ```__version__``` tag...

**Comment by rth** [comment]
> anything else we need to do for v1.4? Only thing I can think of is updating the __version__ tag...

Sounds good to me. Maybe it be also check that the [release notes]( https://github.com/bsmurphy/PyKrige/blob/master/release_notes.md) are up to date.. Actually, maybe it could be worth making that a section of the documentation as well, what do you think?

**Comment by rth** [comment]
I pushed a pre-release [v1.4.0rc1 to PyPi](https://pypi.org/project/PyKrige/1.4.0rc1/), it can be installed with,
```
pip install PyKrige==1.4.0rc1
```

@bsmurphy  @basaks Please let me know if you have any comments or other things that should be added. 

The links to the stable documentation in the Readme are currently broken because we need to make an actual release with a v1.4.0 tag on github for it to work. The latest version works fine http://pykrige.readthedocs.io/en/latest/ and is automatically triggered for new commits. So the issue with readthedocs is solved.
